test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1157 @@
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 12,
+  "result": {
+    "tokenId": "0.0.541",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Wipe Key:**
+
+- The key which can wipe the token's balance from an account.
+
+| Test no | Name                                                                                                            | Input                                                                                                                      | Expected response                                                                                              | Implemented (Y/N) |
+|---------|-----------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates a token with a valid ED25519 public key as its wipe key                                                 | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, wipeKey=<VALID_ED25519_PUBLIC_KEY>          | The token creation succeeds and the token has the new ED25519 public key as its wipe key.                      | N                 |
+| 2       | Creates a token with a valid ECDSAsecp256k1 public key as its wipe key                                          | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, wipeKey=<VALID_ECDSA_SECP256K1_PUBLIC_KEY>  | The token creation succeeds and the token has the new ECDSAsecp256k1 public key as its wipe key.               | N                 |
+| 3       | Creates a token with a valid ED25519 private key as its wipe key                                                | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, wipeKey=<VALID_ED25519_PRIVATE_KEY>         | The token creation succeeds and the token has the new corresponding ED25519 public key as its wipe key.        | N                 |
+| 4       | Creates a token with a valid ECDSAsecp256k1 private key as its wipe key                                         | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, wipeKey=<VALID_ECDSA_SECP256K1_PRIVATE_KEY> | The token creation succeeds and the token has the new corresponding ECDSAsecp256k1 public key as its wipe key. | N                 |
+| 5       | Creates a token with a valid KeyList of ED25519 and ECDSAsecp256k1 private and public keys as its wipe key      | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, wipeKey=<VALID_KEYLIST>                     | The token creation succeeds and the token has the new KeyList as its wipe key.                                 | N                 |
```

Comment:
Why are some arguments concrete like: 
`name = "testname"` and then we have a an argument like `treasuryAccountId=<OPERATOR_ACCOUNT_ID>`
Shouldn't we stick to one convention?

Reply:
My thought was arguments in `<>` brackets will not necessarily always be the same, where as concrete arguments will always be that. For example, `<OPERATOR_ACCOUNT_ID>` could be different if for whatever reason we want to test with a different operator, but it would still represent that same idea. Same thing for any sort of `<VALID_KEY>`, that will be determined based on the returned value of `generateKey`.